src/ontology/mondo.Makefile

```diff
@@ -268,8 +269,45 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts (Aug 2023) pipeline
+# - https://github.com/monarch-initiative/mondo-ingest/issues/273
+.PHONY: address-medgen-conflicts-aug2023
+address-medgen-conflicts-aug202x3: address-medgen-conflicts-aug2023-deletes address-medgen-conflicts-aug2023-adds
+
+# - MedGen conflicts: dependencies
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+# - MedGen conflicts: analysis
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	-diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/medgen_conflicts_removals_diff_analysis.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+tmp/report-qc-medgen-conflicts-adds-diff.tsv: $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv
```

Comment:
## Megan's 278 additions absent from ingest
@matentzn Added script w/ goal to analyze any xref addition discrepancies between what is coming from our ingest and what's coming from curated conflicts, with the main purpose be to see if there's anything recommended from curation that is not coming in through the ingest.

The result is 35,567 additions coming in through the ingest that were not among Megan's recommended replacements. A high number is to be expected as she was just curating some bad Mondo xrefs and making recommended replacements.

However, Megan made 278 recommendations that did not appear in the ingest. I only checked 1 case, and it is found in the FTP's `MedGenIDMappings.txt`; it was just not mapped explicitly to Mondo. I expect the other terms will also appear. My guess here is that Megan's ad hoc process to create these recommendations for us involved a lot of manual work and was decoupled from whatever process generates `MedGenIDMappings.txt`. Perhaps she derived these mappings via proxy (in the one case I looked at, the term was mapped to an OMIM term).

**Suggested action**
I suggest we bring this list to Megan and, ideally, for each term, evaluate whether (a) they should update `MedGenIDMappings.txt` w/ the mapping, or (b) see if she made the recommendation in error.

[report-qc-medgen-conflicts-adds-diff.tsv.zip](https://github.com/monarch-initiative/mondo/files/12564166/report-qc-medgen-conflicts-adds-diff.tsv.zip)

Reply:
@joeflack4 can you turn this into an issue link it to this comment and resolve it here. Since these are "false negatives" xrefs (i.e. missing mappings) and not "false positive" mappings (wrong mappings), lets deal with these in a second step.

Reply:
Sounds good! Here's the issue created: https://github.com/monarch-initiative/mondo-ingest/issues/366

Also, I went ahead and let Megan know about this. You can also see that and her response: [here](https://github.com/monarch-initiative/mondo-ingest/issues/273#issuecomment-1712658861)